chemprop/nn/agg.py

```diff
@@ -47,15 +47,15 @@ def forward(self, H: Tensor, batch: Tensor) -> Tensor:
         Parameters
         ----------
         H : Tensor
-            A tensor of shape ``V x d`` containing the batched node-level representations of ``n``
+            A tensor of shape ``V x d`` containing the batched node-level representations of ``b``
```

Comment:
Did we define `b` anywhere?

Reply:
It is defined [here](https://github.com/chemprop/chemprop/blob/5aeac9eff6c32ff6416df0ed3a03cad94df22cb3/chemprop/nn/message_passing/base.py#L158) in message_passing base.py. We could define `b` here and probably should, but at the same time we don't define what `V` and `d` are. Additionally there are many docstrings that use variables without defining them each time. I think it would be better to collect all the variable definitions into a single reference like what some textbooks do, but that is fairly big task.